apps/teams-test-app/e2e-test-data/nestedAppAuth.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "nestedAppAuth",
+  "platforms": "*",
```

Comment:
Validating whether this '*' breaks the pipeline on mobile platforms if the UI tests have not yet been implemented there.

Reply:
No, it will not.

Reply:
Closing this validation question as it will not cause any breakage. Thanks @KangxuanYe